backend/api/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+class BusinessSerializer(serializers.Serializer):
+    camis = serializers.CharField()
+    dba = serializers.CharField(required=False)
+    boro = serializers.CharField(required=False)
+    street = serializers.CharField(required=False)
+    building = serializers.CharField(required=False)
+    zipcode = serializers.CharField(required=False)
+    cuisine_description = serializers.CharField(required=False)
```

Comment:
How do i deal with missing values in a dataset?  If i want the dba ('doing business as' aka business name) to be required, will this no longer break if i add allow blank/null options to validate? 
http://www.django-rest-framework.org/api-guide/fields/#charfield